chia/wallet/wallet_state_manager.py

```diff
@@ -874,6 +874,7 @@ async def new_coin_state(
                         "pool_wallet",
                     )
                     coin_added = launcher_spend.additions()[0]
+                    assert len(launcher_spend.additions()) == 1
```

Comment:
Not needed but just a sanity check, according to Quex this is always true.

Reply:
Can we make this not run `additions()` twice?  It runs CLVM which is often slow.